### ISSUE TITLE
Can't train the baseline model with GPU

### ISSUE THREAD

**Issue by RakeshJarupula** [radiantearth/Nasa_harvest_field_boundary_competition#1]
I am not able to train the baseline model with GPU enabled. Below is the error. I think it's the problem with library versions. Training works fine on cpu.
```
InternalError: Exception encountered when calling layer "bn_data" (type BatchNormalization).

cuDNN launch failure : input shape ([4,256,256,3]) [Op:FusedBatchNormV3]

Call arguments received:
  • inputs=tf.Tensor(shape=(4, 256, 256, 3), dtype=float32)
  • training=True
```

**Comment by guidorice** [comment]
@RakeshJarupula thanks for the feedback on this baseline model. We will try to reproduce the error on GPU and followup with you after that.

cc @magabenza @DanielNwaeze

**Comment by guidorice** [comment]
@RakeshJarupula one additional question: are you running on a GPU in Google Colab, or on your local dev environment?

**Comment by RakeshJarupula** [comment]
> @RakeshJarupula one additional question: are you running on a GPU in Google Colab, or on your local dev environment?

I am using Kaggle platform for training the model. And installed "colab" requirements.

**Comment by DanielNwaeze** [comment]
Hi @RakeshJarupula 
I tried replicating this issue on Kaggle and it seems like their GPUs don't just have the memory resources to support this, they suggested upgrading to Google Cloud AI Platform.

I'm not certain if it's a library issue, I did try multiple library versions but none of them worked. This was one of the reasons we chose to use Colab's platform as it had enough resources and compatibility to support model training with GPU, also for uniformity.
Some solutions that could've helped might be upgrading the CUDA version, but Kaggle doesn't support this.

I think overall, it's clear that using Kaggle default GPU isn't very suited here, and as this is outside the scope of the work we did, I'm afraid I cannot provide further assistance on this.

cc @guidorice @pieschker
